usaspending_api/awards/migrations/0013_add_unmanaged_tables.py

```diff
@@ -0,0 +1,387 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2018-01-12 18:47
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0012_upper_index_fain_uri_awards'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AwardCategory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('type_code', models.TextField(blank=True)),
+                ('type_name', models.TextField(blank=True)),
+            ],
+            options={
+                'db_table': 'award_category',
+                'managed': False,
+            },
+        ),
+        migrations.CreateModel(
```

Comment:
Will there be issues with django "creating" but not managing these materialized views?

Reply:
There shouldn't be a problem with this. It's just doing some accounting by having it in the migrations. Doesn't affect anything outside of the code.